src/setup/rgw.py

```diff
@@ -129,89 +131,43 @@ def config(conf: MappedConf):
             Rgw._update_rgw_config_with_endpoints(conf, service_endpoints, instance)
             instance = instance + 1
 
+        # Before user creation,Verify backend store value=motr in rgw config file.
+        Rgw._verify_backend_store_value(conf)
+
+        # Read Motr HA(HAX) endpoint from data pod using hctl fetch-fids and update in config file
+        # Use remote hax endpoint running on data pod which will be available during rgw
+        # config phase since data pod starts before server pod.
+        current_data_node = socket.gethostname().replace('server', 'data')
```

Comment:
Add another comment
\# Try data pod of same node first & if it doesnt work, try HAX endpoints from all data pods in cluster

Reply:
done